pyiron_continuum/mesh.py

```diff
@@ -107,6 +119,7 @@ def __init__(
             self,
             bounds: Union[float, int, BoundsList, np.ndarray] = 1,
             divisions: Union[int, List[int], np.ndarray] = 1,
+            accuracy: int = 2,
```

Comment:
It's written default is 4 below

Reply:
good eye. Will let the tests finish then push the patch.